package.json

```diff
@@ -19,7 +19,7 @@
     "cbuild": "cbuild",
     "dump-lib": "dump-em-lib dist/em-api.js",
     "lint": "tslint -c src/tslint.json src/*.ts src/em/*.ts",
-    "prepublish": "npm run lint && tsc -p src/em && tsc -p src && cbuild -x -v -s dist/bundle/em/em-api.js -o dist/em-api.js",
+    "prepublish": "tsc -p src/em && tsc -p src && cbuild -x -v -s dist/bundle/em/em-api.js -o dist/em-api.js && npm run lint",
```

Comment:
I guess reversing would be best, so it checks style before typing issues.

Installing from a repo also doesn't install devDependencies so the only way to get that working is to put compiled `.js` files in the repo. I'll do a release tomorrow to avoid problems.